frontend/src/components/Contact/Contact.jsx

```diff
@@ -29,7 +29,9 @@ const Contact = () => {
 
   const submit = e => {
     e.preventDefault();
-    agent.Contact.send({ user: userType, name, firstName, email, reason, message })
+    agent.Contact.send(
+      { user: userType, name, firstName, email, reason, message, navigator: navigator.appVersion },
```

Comment:
I wonder if `userAgent` is better than `appVersion`. With `userAgent` you can use tools to decode it (e.g. https://developers.whatismybrowser.com/useragents/parse/)